content/en/docs/ambient/usage/ambient-and-networkpolicy/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Ambient and Kubernetes NetworkPolicy
+description: Understanding how CNI-enforced L4 Kubernetes NetworkPolicy interacts with ambient.
```

Comment:
we don't usually want people saying just "ambient" to refer to "Istio in ambient mode" but maybe it's time to realise that everyone does and we should

Reply:
It's likely "just me" and not "everyone" - I'll just say "ambient mode" here.

Reply:
Do we have any real guidelines for how to refer to this? Ambient mode vs Ambient mesh vs Ambient data plane vs Istio X vs just Ambient.

In this case "Understanding how CNI-enforced L4 Kubernetes NetworkPolicy interacts with the Ambient data plane" makes sense to me for instance.

Reply:
I changed it to `istio ambient mode`

Reply:
> Do we have any real guidelines for how to refer to this? Ambient mode vs Ambient mesh vs Ambient data plane vs Istio X vs just Ambient.

Just to use a lower case "a" 😆 

https://istio.io/latest/docs/reference/glossary/#ambient is the current official status